pydap/util/http.py

```diff
@@ -25,7 +25,8 @@ def request(url):
     """
```

Comment:
Roberto moved most of the networking code over to `src/pydap/net.py`, which basically just defines a function `GET()` which wraps a `webob` request. If you want to modify SSL validation or proxy settings, you probably want to do it there, with `webob` or with `httplib` (that's essentially the software stack from the top down).